First round of API v2 updates. (#2340)
XRPLF/xrpl-dev-portal: content/references/http-websocket-apis/public-api-methods/account-methods/account_info.md

| `ledger_hash`  | String               | No        | A 20-byte hex string for the ledger version to use. (See [Specifying Ledgers][]) |
| `ledger_index` | Number or String     | No        | The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. (See [Specifying Ledgers][]) |
| `queue`        | Boolean              | No        | If `true`, return stats about [queued transactions](transaction-queue.html) sent by this account. Can only be used when querying for the data from the current open ledger. Not available from servers in [Reporting Mode][]. |

> | `signer_lists` | Boolean              | No        | [API v1][]: If `true`, return any [SignerList objects](signerlist.html) associated with this account.<br>[API v2][]: Identical to v1, but also returns an `invalidParams` error if a non-boolean value is used. |

Reviewer: Avoid HTML in tables for Redocly compatibility; also, I suggest rephrasing to active voice.

```suggestion
| `signer_lists` | Boolean              | No        | [API v1][]: If `true`, return any [SignerList objects](signerlist.html) associated with this account. [API v2][] returns an `invalidParams` error if you provide a non-boolean value. |
```

|:-----------------------|:--------|:------------------------------------------|
| `account_data`         | Object  | The [AccountRoot ledger object](accountroot.html) with this account's information, as stored in the ledger. |
| `account_flags`        | Object  | The account's flag statuses (see below), based on the `Flags` field of the account. [New in: rippled 1.11.0][] |

> | `signer_lists`         | Array   | [API v1][]: _(Omitted unless the request specified `signer_lists` and at least one SignerList is associated with the account.)_ Array of [SignerList ledger objects](signerlist.html) associated with this account for [Multi-Signing](multi-signing.html). Since an account can own at most one SignerList, this array must have exactly one member if it is present. The field is nested under `account_data`.<br>[API v2][]: Identical to API v1, but the field is returned in the root reponse instead. [Clio](https://github.com/XRPLF/clio) implements the API v2 behavior. |

Reviewer: As above, avoid HTML in tables for Redocly compatibility. Also, I spotted a typo.

```suggestion
| `signer_lists`         | Array   | _(Omitted unless the request specified `signer_lists` and at least one SignerList is associated with the account.)_ Array of [SignerList ledger objects](signerlist.html) associated with this account for [Multi-Signing](multi-signing.html). Since an account can own at most one SignerList, this array must have exactly one member if it is present. In [API v1][], the field is nested under `account_data`. In [API v2][], the field is returned in the root response instead. [Clio](https://github.com/XRPLF/clio) implements the API v2 behavior in all cases. |
```
